pom.xml

```diff
@@ -592,6 +572,8 @@
         </os>
       </activation>
       <properties>
+        <!-- Only fail code coverage on non-windows machines -->
+        <jacoco.haltOnFailure>true</jacoco.haltOnFailure>
```

Comment:
Rather than exclude some classes from measurement due to tests not running on windows, don't run coverage on windows.